receiver/awscontainerinsightreceiver/internal/ecsInfo/ecsinfo_test.go

```diff
@@ -1,16 +1,16 @@
-//Copyright  OpenTelemetry Authors
```

Comment:
These changes look unrelated

Reply:
Yes. But if not change like this, `gocritic` will check those comment as not passed, and can't be escaped by nolint , so I directly modify those comments to correct format.

Reply:
Thanks! That makes sense, we can keep it then